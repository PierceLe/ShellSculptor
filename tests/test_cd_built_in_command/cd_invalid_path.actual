>> cd ~/hehehehehehehehe
cd: no such file or directory: /Users/piercele/hehehehehehehehe

