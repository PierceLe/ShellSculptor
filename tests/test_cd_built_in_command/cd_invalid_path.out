>> cd ~/hehehehehehehehe
cd: no such file or directory: /home/hehehehehehehehe

